app/src/main/java/com/wafflestudio/siksha2/preferences/SikshaPrefObjects.kt

```diff
@@ -40,4 +41,13 @@ class SikshaPrefObjects @Inject constructor(
             serializer,
             String::class.java
         )
+
+    val oAuthProvider: Preference<OAuthProvider> =
+        Preference(
+            "oAuthProvider",
+            OAuthProvider.KAKAO,
+            sharedPreferences,
+            serializer,
+            OAuthProvider::class.java
+        )
```

Comment:
Nullable Preference 가 없어서... ㅠㅠ 
위에 주석 정도 남겨놓는게 좋을 것 같아요
```suggestion
        // TODO: nullable preference 만들어서 초기값 null 로 두기, 임시로 kakao 로 설정
        Preference(
            "oAuthProvider",
            OAuthProvider.KAKAO,
            sharedPreferences,
            serializer,
            OAuthProvider::class.java
        )
```

Reply:
주석 추가했습니다!